[2022-02-09 07:44:05,042] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: taxi_zones_data_ingestion_gcs_dag.local_to_gcs_task manual__2022-02-09T07:43:58.429216+00:00 [queued]>
[2022-02-09 07:44:05,054] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: taxi_zones_data_ingestion_gcs_dag.local_to_gcs_task manual__2022-02-09T07:43:58.429216+00:00 [queued]>
[2022-02-09 07:44:05,054] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 07:44:05,054] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 07:44:05,054] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 07:44:05,071] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-02-09 07:43:58.429216+00:00
[2022-02-09 07:44:05,077] {standard_task_runner.py:52} INFO - Started process 1105 to run task
[2022-02-09 07:44:05,082] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'taxi_zones_data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2022-02-09T07:43:58.429216+00:00', '--job-id', '350', '--raw', '--subdir', 'DAGS_FOLDER/taxi_zones_data_ingest_gcs_dag.py', '--cfg-path', '/tmp/tmpb759pt68', '--error-file', '/tmp/tmpkhyy8qhp']
[2022-02-09 07:44:05,083] {standard_task_runner.py:77} INFO - Job 350: Subtask local_to_gcs_task
[2022-02-09 07:44:05,152] {logging_mixin.py:109} INFO - Running <TaskInstance: taxi_zones_data_ingestion_gcs_dag.local_to_gcs_task manual__2022-02-09T07:43:58.429216+00:00 [running]> on host 3d1f29ca65cd
[2022-02-09 07:44:05,218] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=taxi_zones_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-09T07:43:58.429216+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-09T07:43:58.429216+00:00
[2022-02-09 07:44:05,773] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 07:44:05,788] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=taxi_zones_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20220209T074358, start_date=20220209T074405, end_date=20220209T074405
[2022-02-09 07:44:05,816] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 07:44:05,862] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
